<launch>
<!-- Launch file for Place recognition example -->

  <arg name="rviz" default="true" />

  <node 
		pkg="btc_desc" 
		type="test_main" 
		name="test_main" 
		output="screen" 
	/>

  <param name="cloud_overlap_thr" type="double" value = "0.5"/>
  <param name="read_bin" type="bool" value="true" />
  <param name="setting_path" type="string" value = "/home/noah/tfm/src/btc_descriptor/config/config_outdoor.yaml"/>
  <param name="pcds_dir" type="string" value = "/home/noah/tfm/images/KITTI/00g/velodyne"/>
  <param name="pose_file" type="string" value = "/home/noah/tfm/src/btc_descriptor/poses/kitti00.txt"/>
  <param name="gt_file" type="string" value = "/home/noah/tfm/src/btc_descriptor/poses/groundtruth00.txt"/>

  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find btc_desc)/rviz_cfg/loop.rviz" />
  </group>

</launch>